### ISSUE TITLE
node becomes not operational for 15 minutes after restart-with-resharding

### ISSUE THREAD

**Issue by yarongilor** [scylladb/scylla#5079]
Installation details
Scylla version (or git commit hash): 3.1.0.rc7-0.20190915.024d1563a
Cluster size: 6
OS (RHEL/CentOS/Ubuntu/AWS AMI): ami-0a01b94dffe4bcc0d

scenario: 50GB longevity. ran a restart with resharding nemesis on node, then after the node is up again, it is not operational, not responding to nodetool commands as well for 15 minutes (perhaps until is was reboot by the following nemesis of restart with repair). the cluster got high latency of ~ 1.5 seconds and low load on this period, as a result.

received events of:
```

(ClusterHealthValidatorEvent Severity.CRITICAL): type=error name=NodesStatus node=longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3 error=Unable to get nodetool status from 'longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3': Encountered a bad command exit code!  Command: 'nodetool -u cassandra -pw cassandra  status '  Exit code: 1  Stdout:    Stderr:  nodetool: Failed to connect to '127.0.0.1:7199' - ConnectException: 'Connection refused (Connection refused)'. | ClusterHealthValidatorEvent, CRITICAL, events, error
-- | --
2019-09-18 12:32:05.150 | (ClusterHealthValidatorEvent Severity.CRITICAL): type=error name=NodeSchemaVersion node=longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3 error=Validate schema version failed. Error: Encountered a bad command exit code!  Command: 'nodetool -u cassandra -pw cassandra  gossipinfo '  Exit code: 1  Stdout:    Stderr:  nodetool: Failed to connect to '127.0.0.1:7199' - ConnectException: 'Connection refused (Connection refused)'. | ClusterHealthValidatorEvent, CRITICAL, events, error
2019-09-18 12:40:09.065 | (ClusterHealthValidatorEvent Severity.CRITICAL): type=error name=NodesStatus node=longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3 error=Unable to get nodetool status from 'longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3': Encountered a bad command exit code!  Command: 'nodetool -u cassandra -pw cassandra  status '  Exit code: 1  Stdout:    Stderr:  nodetool: Failed to connect to '127.0.0.1:7199' - ConnectException: 'Connection refused (Connection refused)'.

```
the following nemesis and failures occurred:
```
[jenkins@aws-eu-builder3 latest]$ grep -v 'ChaosMonkey on target' sct_9-18.log | grep -i 'found coredump\|Set current_disruption\|segmentation\|<<<<<<<<<\|p:ERROR > Traceback' 
< t:2019-09-18 02:55:23,098 f:nemesis.py      l:595  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: <<<<<<<<<<<<<Finished random_disrupt_method destroy_data_then_repair
< t:2019-09-18 03:15:02,621 f:nemesis.py      l:595  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: <<<<<<<<<<<<<Finished random_disrupt_method nodetool_enospc
< t:2019-09-18 03:34:41,668 f:nemesis.py      l:595  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: <<<<<<<<<<<<<Finished random_disrupt_method nodetool_enospc
< t:2019-09-18 03:53:38,492 f:nemesis.py      l:338  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Set current_disruption -> ModifyTablePropertiesDclocalReadRepairChance Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-5 [34.248.223.143 | 10.0.63.76] (seed: False)
< t:2019-09-18 03:54:08,948 f:nemesis.py      l:595  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: <<<<<<<<<<<<<Finished random_disrupt_method modify_table
< t:2019-09-18 04:13:20,674 f:nemesis.py      l:338  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Set current_disruption -> TruncateMonkey Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3 [34.247.162.93 | 10.0.69.237] (seed: False)
< t:2019-09-18 04:15:29,804 f:nemesis.py      l:1071 c:sdcm.nemesis         p:ERROR > Traceback (most recent call last):
< t:2019-09-18 04:35:06,021 f:nemesis.py      l:338  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Set current_disruption -> ModifyTablePropertiesCompaction Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-5 [34.248.223.143 | 10.0.63.76] (seed: False)
< t:2019-09-18 04:35:34,818 f:nemesis.py      l:595  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: <<<<<<<<<<<<<Finished random_disrupt_method modify_table
< t:2019-09-18 04:54:58,207 f:nemesis.py      l:338  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Set current_disruption -> TerminateAndReplaceNode Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-4 [34.240.105.72 | 10.0.159.212] (seed: False)
< t:2019-09-18 06:40:22,124 f:cluster.py      l:833  c:sdcm.cluster         p:DEBUG > Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-6 [52.214.107.136 | 10.0.42.134] (seed: False): Found coredump file: /var/lib/systemd/coredump/core.scylla.996.fda02d2cfd394d57ad5130894d4813bc.64535.1568788699000000
< t:2019-09-18 06:45:46,572 f:nemesis.py      l:1071 c:sdcm.nemesis         p:ERROR > Traceback (most recent call last):
< t:2019-09-18 07:02:23,258 f:nemesis.py      l:338  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Set current_disruption -> RestartThenRepairNode Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-6 [52.214.107.136 | 10.0.42.134] (seed: False)
< t:2019-09-18 12:07:47,684 f:nemesis.py      l:595  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: <<<<<<<<<<<<<Finished random_disrupt_method restart_then_repair_node
< t:2019-09-18 12:24:15,634 f:nemesis.py      l:338  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Set current_disruption -> RestartNode with resharding Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3 [34.247.162.93 | 10.0.69.237] (seed: False)
< t:2019-09-18 12:31:04,446 f:nemesis.py      l:1071 c:sdcm.nemesis         p:ERROR > Traceback (most recent call last):
< t:2019-09-18 12:44:33,005 f:nemesis.py      l:338  c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.ChaosMonkey: Set current_disruption -> RestartThenRepairNode Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3 [34.247.162.93 | 10.0.69.237] (seed: False)

```
logs:
https://cloudius-jenkins-test.s3.amazonaws.com/5979c9a0-a1c1-4601-a2c3-63f1850766d9/longevity-tls-50gb-4d-3-1-loader-set-5979c9a0.zip
https://cloudius-jenkins-test.s3.amazonaws.com/5979c9a0-a1c1-4601-a2c3-63f1850766d9/longevity-tls-50gb-4d-3-1-monitor-set-5979c9a0.zip
https://cloudius-jenkins-test.s3.amazonaws.com/5979c9a0-a1c1-4601-a2c3-63f1850766d9/monitoring_data_stack_branch-2.4_3.1.tar.gz

 https://cloudius-jenkins-test.s3.amazonaws.com/5979c9a0-a1c1-4601-a2c3-63f1850766d9/longevity-tls-50gb-4d-3-1-db-cluster-5979c9a0.zip

job log: scratch.scylladb.com/qa_issues/sct_08_42_22_09_2019.log.gz

**Comment by slivne** [comment]
We need metrics, node ip and time range

We have a limit  (via the service file using systemd) on node comming up of
900 seconds - 15 minutes. So a node that dors not boot at that period will
be killed and may be started again.

We have seen such cases when replaying of commitlogs took to long

Need to check if the scylla process was killed and started by checking the
journal/pid

**Comment by yarongilor** [comment]
node ip: 10.0.69.237
monitor: http://52.212.4.91:3000

**Comment by slivne** [comment]
the log file for sct seems bad

```
< t:2019-09-18 12:24:15,651 f:nemesis.py      l:340  c:sdcm.nemesis         p:INFO  > sdcm.nemesis.ChaosMonkey: RestartNode with resharding Node longevity-tls-50gb-4d-3-1-db-node-5979c9a0-3 [34.247.162.93 | 10.0
.69.237] (seed: False)
< t:2019-09-18 12:24:15,653 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:06:31+00:00  ip-10-0-42-134 !WARNING | scylla: [shard 11] gossip - unknown endpoint 10.0.159.212
< t:2019-09-18 12:24:15,654 f:remote.py       l:642  c:sdcm.remote          p:INFO  > RemoteCmdRunner [centos@10.0.105.154]: insert,    116888605,     481,     481,     481,     6.5,     3.8,    28.9,    47.1,  
  80.3,    80.3,140065.0,  0.05369,      0,      0,       0,       0,       0,       0
< t:2019-09-18 12:24:15,654 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:06:01+00:00  ip-10-0-169-31 !WARNING | scylla: [shard 2] gossip - unknown endpoint 10.0.159.212
< t:2019-09-18 12:24:15,654 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T07:31:30+00:00  ip-10-0-114-220 !INFO    | scylla: [shard 0] stream_session - [Stream #54d93fa0-d9e6-11e9-b8b2-000
000000003] Start sending ks=system_traces, cf=sessions_time_idx, estimated_partitions=0, with new rpc streaming
< t:2019-09-18 12:24:15,655 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:12:01+00:00  ip-10-0-63-76 !WARNING | scylla: [shard 7] gossip - unknown endpoint 10.0.159.212
< t:2019-09-18 12:24:15,655 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:07:01+00:00  ip-10-0-69-237 !WARNING | scylla: [shard 7] gossip - unknown endpoint 10.0.159.212
< t:2019-09-18 12:24:15,651 f:remote.py       l:642  c:sdcm.remote          p:INFO  > RemoteCmdRunner [centos@10.0.229.230]: read2,      38696093,     168,     168,     168,    12.8,     5.0,    45.9,    72.3,  
  83.2,    83.2,140041.0,  0.06789,      0,      0,       0,       0,       0,       0
< t:2019-09-18 12:24:15,655 f:sct_events.py   l:530  c:sdcm.sct_events      p:INFO  > {"id":542,"message":"Annotation added"}
< t:2019-09-18 12:24:15,656 f:remote.py       l:642  c:sdcm.remote          p:INFO  > RemoteCmdRunner [centos@10.0.172.174]: read3,      30791921,     117,     117,     117,    13.0,     4.3,    42.4,    62.0,  
 150.9,   150.9,140029.0,  0.08230,      0,      0,       0,       0,       0,       0
< t:2019-09-18 12:24:15,656 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:12:31+00:00  ip-10-0-188-94 !WARNING | scylla: [shard 12] gossip - unknown endpoint 10.0.159.212
< t:2019-09-18 12:24:15,657 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:06:31+00:00  ip-10-0-42-134 !WARNING | scylla: [shard 11] gossip - unknown endpoint 10.0.159.212
< t:2019-09-18 12:24:16,038 f:remote.py       l:642  c:sdcm.remote          p:INFO  > RemoteCmdRunner [centos@10.0.105.154]: read1,      38962362,     185,     185,     185,    12.3,     4.8,    48.0,    53.7,  
  82.3,    82.3,140065.0,  0.05369,      0,      0,       0,       0,       0,       0
< t:2019-09-18 12:24:16,042 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:06:01+00:00  ip-10-0-169-31 !WARNING | scylla: [shard 2] gossip - unknown endpoint 10.0.159.212
< t:2019-09-18 12:24:16,137 f:remote.py       l:642  c:sdcm.remote          p:INFO  > RemoteCmdRunner [centos@10.0.172.174]: total,     241900019,    1621,    1621,    1621,     6.8,     4.9,    10.2,    23.8,  
 311.7,  1126.2,160840.0,  0.01873,      0,      0,       0,       0,       0,       0
< t:2019-09-18 12:24:16,138 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T07:31:30+00:00  ip-10-0-114-220 !INFO    | scylla: [shard 8] stream_session - [Stream #54d93fa0-d9e6-11e9-b8b2-000
000000003] Start sending ks=system_traces, cf=sessions_time_idx, estimated_partitions=0, with new rpc streaming
```

we have restart and resharding running in parallel to streaming - or we are making a mess in the  appending of log files

```
2019-09-18T06:06:01+00:00
2019-09-18T07:31:30+00:00
```

in addition I can't find an indication that scylla was shutdown and started

aside of this
```
< t:2019-09-18 12:25:50,407 f:remote.py       l:108  c:sdcm.remote          p:INFO  > RemoteCmdRunner [centos@10.0.69.237]: Command "sudo systemctl stop scylla-server.service" finished with status 0
< t:2019-09-18 12:26:00,868 f:remote.py       l:213  c:sdcm.remote          p:DEBUG > RemoteCmdRunner [centos@10.0.69.237]: Running command "sudo systemctl start scylla-server.service"...
```

maybe its because journal supresses messages all the time
 

```
< t:2019-09-18 12:16:20,740 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:07:00+00:00  ip-10-0-69-237 !INFO    | journal: Suppressed 174319 messages from /system.slice/scylla-server.ser
vice
< t:2019-09-18 12:17:01,786 f:cluster.py      l:1231 c:sdcm.cluster         p:DEBUG > 2019-09-18T06:07:00+00:00  ip-10-0-69-237 !INFO    | scylla: [shard 10] compaction - Compacted 2 sstables to [/var/lib/scylla
/data/keyspace_lz4/standard1-50c11310d89811e981fc00000000000c/mc-63920-big-Data.db:level=0, ]. 521268 bytes to 513592 (~98% of original) in 893ms = 0.55MB/s. ~2304 total partitions merged to 2226.
```
```

**Comment by slivne** [comment]
I tried to access the monitor: http://52.212.4.91:3000 its seems to be down - @yarongilor can you please bring it back up - I'll try to see if I can extract something from the metrics

**Comment by slivne** [comment]
@yarongilor ping

**Comment by slivne** [comment]
Closing this issue - stale - if this reproduces please create a new issue
